react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+
+const buttons = [
+  { label: 'I', style: 'red' },
+  { label: 'Know', style: 'green' },
+  { label: 'React', style: 'blue' },
+];
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <CustomButton buttons={buttons} />
```

Comment:
Per the instructions "Replace the code in App.js with code that renders 3 instances of your CustomButton in a div, passing a text prop to the first with value "I", the second with "know", and the third with "React!""

This would be something closer to:

export default function App() {
  return (
    <div>
      <CustomButton text="I" color="red" />
      <CustomButton text="know" color="green" />
      <CustomButton text="React!" color="lightblue" />
    </div>
  );
}